code-of-conduct-IT.md

```diff
@@ -0,0 +1,42 @@
+## Introduzione
+
+Ci impegniamo a garantire a tutti un'esperienza senza molestie e non le tolleriamo in nessuna forma. Ti chiediamo di tenere in considerazione gli altri e di comportarti professionalmente e con rispetto verso tutti gli altri partecipanti. Questo codice e le relative pratiche si applicano anche ai comportamenti inaccettabili che si verificano al di fuori dell'ambito delle attività comunitarie, in qualsiasi luogo della comunità - in linea e di persona - così come in tutte le comunicazioni dirette e dovunque tale comportamento possa avere influenzare negativamente la sicurezza e il benessere dei membri della comunità. Gli espositori, oratori, sponsor, il personale e tutti gli altri partecipanti agli eventi organizzati da Docker, Inc (DockerCon, meetups, riunioni di utenti) o presso le strutture sono soggetti a queste linee guida comunitarie e codice di condotta.
```

Comment:
- `in linea`-> io lascerei online e di persona.
- `possa avere influenzare` negativamente -> possa influenzare negativamente
- `Docker, Inc` -> Docker, Inc.